ProcessesApi.Tests/V1/Services/SoleToJointServiceTests.cs

```diff
@@ -647,5 +657,89 @@ public async Task ProcessUpdatedEventIsRaisedWhenDocumentsAppointmentIsReschedul
         }
 
         #endregion Reschedule documents appointment
+
+        #region Review Documents
+
+        [Theory]
+        [InlineData(SoleToJointStates.DocumentsRequestedDes)]
+        [InlineData(SoleToJointStates.DocumentsRequestedAppointment)]
+        [InlineData(SoleToJointStates.DocumentsAppointmentRescheduled)]
+        public async Task ProcessStateIsUpdatedToDocumentChecksPassed(string initialState)
+        {
+            // Arrange
+
+            var process = CreateProcessWithCurrentState(initialState);
+
+            var formData = _reviewDocumentCheckPass;
+            var triggerObject = CreateProcessTrigger(process,
+                                                     SoleToJointPermittedTriggers.ReviewDocuments,
+                                                     formData);
+            // Act
+            await _classUnderTest.Process(triggerObject, process, _token).ConfigureAwait(false);
+
+            // Assert
+            CurrentStateShouldContainCorrectData(process,
+                                                 triggerObject,
+                                                 SoleToJointStates.DocumentChecksPassed,
+                                                 new List<string> { /*Add new state here when implemented*/ });
+            process.PreviousStates.LastOrDefault().State.Should().Be(initialState);
+        }
+
+        [Theory]
+        [InlineData(SoleToJointStates.DocumentsRequestedDes)]
+        [InlineData(SoleToJointStates.DocumentsRequestedAppointment)]
+        [InlineData(SoleToJointStates.DocumentsAppointmentRescheduled)]
+        public async Task ProcessUpdatedEventIsRaisedWhenDocumentsChecksPassed(string initialState)
```

Comment:
This doesn't need to be a separate test now :)
Just call ThenProcessUpdatedEventIsRaised() at the end of the previous test, it saves us from repeating the test again.